tests/gcm/test_arrow_strength.py

```diff
@@ -26,97 +26,26 @@ def preserve_random_generator_state():
 
 @flaky(max_runs=5)
 def test_given_kl_divergence_attribution_func_when_estimate_arrow_strength_then_returns_expected_results():
-    causal_model = ProbabilisticCausalModel(nx.DiGraph([('X1', 'X2'), ('X0', 'X2')]))
-    causal_model.set_causal_mechanism('X1', ScipyDistribution(stats.norm, loc=0, scale=1))
-    causal_model.set_causal_mechanism('X0', ScipyDistribution(stats.norm, loc=0, scale=1))
-    causal_model.set_causal_mechanism('X2', AdditiveNoiseModel(prediction_model=create_linear_regressor()))
-
-    X0 = np.random.normal(0, 1, 1000)
-    X1 = np.random.normal(0, 1, 1000)
-
-    test_data = pd.DataFrame({'X0': X0,
-                              'X1': X1,
-                              'X2': 3 * X0 + X1 + np.random.normal(0, 0.2, X0.shape[0])})
-    fit(causal_model, test_data)
-
-    causal_strengths = arrow_strength(causal_model,
+    causal_strengths = arrow_strength(_create_causal_model(),
                                       'X2',
                                       difference_estimation_func=estimate_kl_divergence_continuous)
     assert causal_strengths[('X0', 'X2')] == approx(2.76, abs=0.4)
     assert causal_strengths[('X1', 'X2')] == approx(1.6, abs=0.4)
 
 
 @flaky(max_runs=5)
-def test_given_continuous_data_with_default_atttribution_func_when_estimate_arrow_strength_then_returns_expected_results():
-    causal_model = ProbabilisticCausalModel(nx.DiGraph([('X1', 'X2'), ('X0', 'X2')]))
-    causal_model.set_causal_mechanism('X1', ScipyDistribution(stats.norm, loc=0, scale=1))
-    causal_model.set_causal_mechanism('X0', ScipyDistribution(stats.norm, loc=0, scale=1))
-    causal_model.set_causal_mechanism('X2', AdditiveNoiseModel(
-        prediction_model=create_linear_regressor()))
-
-    X0 = np.random.normal(0, 1, 1000)
-    X1 = np.random.normal(0, 1, 1000)
+def test_given_continuous_data_with_default_attribution_func_when_estimate_arrow_strength_then_returns_expected_results():
+    causal_strengths = arrow_strength(_create_causal_model(), 'X2')
 
-    test_data = pd.DataFrame({'X0': X0,
-                              'X1': X1,
-                              'X2': 3 * X0 + X1 + np.random.normal(0, 0.2, X0.shape[0])})
-    fit(causal_model, test_data)
-
-    causal_strengths = arrow_strength(causal_model, 'X2', tolerance=10 ** -4)
+    # By default, the strength is measure with respect to the variance.
     assert causal_strengths[('X0', 'X2')] == approx(9, abs=0.5)
     assert causal_strengths[('X1', 'X2')] == approx(1, abs=0.2)
 
 
-@flaky(max_runs=5)
-def test_arrow_strength_with_squared_mean_difference():
```

Comment:
Why are you removing this test and the one below? (Please add in the commit message)

Reply:
Added it to the commit message and PR description.